Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/fair.h

```diff
@@ -99,6 +99,8 @@ namespace internal {
 
   @pre `is_triangle_mesh(tmesh)`
 
+  @warning This function involves square root computations that will be done approximately.
+
```

Comment:
I do not think that is exact. That is not "square root computations".

I suggest:
> This function involves linear algebra, that is compute using a non-exact floating-point arithmetic.

or something similar.